crates/apollo-parser/src/parser/mod.rs

```diff
@@ -232,11 +228,30 @@ impl Parser {
         }
     }
 
+    /// gets the next token from the lexer
+    fn next_token(&mut self) -> Option<Token> {
```

Comment:
Do you want this entirely local to the parser struct or as `pub(crate) fn`? (just want to double check